src/main/views/resolve-before-claim.njk

```diff
@@ -0,0 +1,47 @@
+{% extends "layout.njk" %}
+
+{% set title = t('Resolving dispute | Money claim') %}
+{% set heading = t('Try to resolve the dispute') %}
+
+{% from "form.njk" import csrfProtection, submitButton, errorSummary %}
+
+{% block content %}
+
+  <div class="grid-row">
+    <div class="column-two-thirds">
+      <p>{{ t('Before you claim you should:') }}</p>
+
+      <ul class="list-bullet form-group">
+        <li>{{ t('talk to the person who owes you') }}</li>
+        <li>{{ t('consider mediation') }}</li>
+      </ul>
+
+      <h2 class="heading-medium">{{ t('Talk to the person who owes you') }}</h2>
+
+      <p>{{ t('Try to resolve the dispute by:') }}</p>
+
+      <ul class="list list-bullet">
+        <li>{{ t('telling them what you believe they owe and why') }}.</li>
+        <li>{{ t('suggesting a timetable for them to pay you back') }}.</li>
+        <li>{{ t('explaining you‘ll make a money claim against them if they don‘t pay') }}.</li>
+      </ul>
+
+      <h2 class="heading-medium">{{ t('Consider mediation') }}</h2>
+
+      <p>{{ t('Mediation is when an impartial person helps both sides to agree - it can be quicker and cheaper than '
+        + ' making a claim.') }}</p>
+      <p>{{ t('It can often be done on the phone. If you don’t want to speak to the person you’re claiming against, '
+        + 'a mediator can speak to you both separately.') }}</p>
+
+      <p>{{ t('<a href="{{ findMediationServiceUri }}">Find a mediation service</a>',
+        { findMediationServiceUri: 'http://civilmediation.justice.gov.uk/' }) | safe }}</p>
+
+      <form novalidate method="post">
```

Comment:
do we need form here?

Reply:
if not - please remove

REQ

Reply:
Form?

Reply:
This is exactly the same as the prototype. It doesn't have a link back but its just meant to be a copy of what we have in the claim journey

Reply:
Don't use the prototype code. They don't adhere to code standards, they use what works to make something look good. if there is no submit button then there is no need for the form